ControlRoomApplication/ControlRoomApplication/GUI/EditTestForm.cs

```diff
@@ -31,5 +31,10 @@ private void splitContainer1_Panel2_Paint(object sender, PaintEventArgs e)
         {
 
         }
+
+        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
+        {
+
```

Comment:
This isn't enough to hold the build back, but I recommend trying to avoid creating empty functions as best as possible. Windows Forms automatically creates them, and they can be a pain to remove, so I would say it's not worth the trouble in your user validation tasks (unless you want to).

This can be more of an end-of-the-semester goal, along with every empty function that I and every other control room team member created.